openreview/conference/invitation.py

```diff
@@ -1437,18 +1452,27 @@ def set_reviewer_recruiter_invitation(self, conference, options = {}):
             'forum': None,
             'replyto': None,
             'readers': {
-                'values': [conference.get_id()]
+                'values-copied': [
+                    conference.get_id(),
+                    '{content.user}'
+                ]
             },
             'signatures': {
                 'values-regex': '\\(anonymous\\)'
             },
             'writers': {
-                'values': []
+                'values': [
```

Comment:
I had to put the anonymous group in the writers field so guest users can edit their response. 

They can not get the response because they are not readers so I don't think this can be a security leak.